python/rmm/_lib/cuda_stream.pyx

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from libc.stdint cimport uintptr_t
+from libcpp cimport bool
+
+
+cdef class CudaStream:
+    """
+    Wraps rmm::cuda_stream (an owning class).
+    """
```

Comment:
A more comprehensive docstring similar to `rmm.DeviceBuffer` would be nice here

Reply:
I'm not sure what to say about `CudaStream` other than it's a RAII wrapper around a CUDA stream. Ideally, no one should interact with `CudaStream` (just `Stream`). Anything specific you'd like to see here?

Reply:
Maybe just add an example to the docstring?